### PR TITLE
Inverted the usebuild event logic

### PR DESCRIPTION
**Pull request by fhamon**
Always set the use-build value in the xml for yes unless ?use-dev is prevent in the url or the use-dev cookie is present.

I also added an opt-out option with ?use-dev=no. It will set the cookie on the client for time() - 3600

### CODE REVIEW

File: workspace/events/event.usebuild.php [hunk 1]
```diff
@@ -34,31 +34,38 @@ public function load()
...
+				$isQuery = false;
+				$isCookie = false;
+			}
+
 			setcookie(
 				$cookieName,
 				'1',
-				2123798400,
+				$cookieTime,
 				__SYM_COOKIE_PATH__,
```
Comment (nitriques) [inline workspace/events/event.usebuild.php]: We should make sure to only keep the .288dev.com part. maybe split on . and remove everything after the last two dots ?

File: workspace/utilities/lib/menu-admin.xsl [hunk 2]
```diff
@@ -26,10 +26,9 @@
...
 					<xsl:choose>
 						<xsl:when test="$debug = true()">
-							<span class="debug">
-								<xsl:text>DEV CODE</xsl:text>
-								<a data-action="full" href="?use-build">
-									I want to test builds
+							<span>
+								<a data-action="full" href="?use-dev">
+									Use build
 								</a>
```
Comment (nitriques) [inline workspace/utilities/lib/menu-admin.xsl]: I think we should restrict this button to developers only.

Comment (nitriques) [inline workspace/utilities/lib/menu-admin.xsl]: Ping @fhamon !

**Comment by fhamon** [comment]
@nitriques pong !

**Comment by nitriques** [comment]
Merged as 04e5ecd5a8

**Pull request closed without merging**